pkg/hubble/build/version_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/mod/semver"
```

Comment:
Looks like we're now explicitly vendoring two semver packages, this one and `github.com/blang/semver` which is already used in other cilium packages. We already use `x/mod/semver` implicitly via transitive dependencies, so I guess it doesn't matter much. But it would be nice to consistently use one or the other across our code base at least.

Reply:
Ah, nice catch. I didn't know we already used `github.com/blang/semver` and figured `x/mod/semver` was already a transitive dependency. I updated the test to use `blang/semver` instead so we keep this consistent across the codebase.